🐙 source-woocommerce: run up-to-date pipeline [2025-05-31]

airbyteio opened the pull request:
# Update source-woocommerce

This PR was autogenerated by running `airbyte-ci connectors --name=source-woocommerce up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


## Operations

- Upgrade the base image to the latest version in metadata.yaml: Successful

- PATCH bump source-woocommerce version to 0.5.5: Successful

- Build source-woocommerce docker image for platform(s) linux/amd64, linux/arm64: Successful

- Get dependency updates: Successful

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful




## Dependency updates

We use [`syft`](https://github.com/anchore/syft) to generate a SBOM for the latest connector version and the one from the PR.
It allows us to spot the dependencies that have been updated at all levels and for all types of dependencies (system, python, java etc.).
Here are the dependencies that have been updated compared to `airbyte/source-woocommerce:latest`.
Keep in mind that `:latest` does not always match the connector code on the main branch. 
It is the latest released connector image when the head commit of this branch was created.

| Type | Name | State | Previous Version | New Version |
|------|------|-------|-------------|------------------|
| python | CacheControl | updated | 0.13.1 | **0.14.2** |
| python | Jinja2 | updated | 3.1.5 | **3.1.6** |
| python | RapidFuzz | updated | 3.10.1 | **3.11.0** |
| python | airbyte-cdk | updated | 6.12.4 | **6.53.0** |
| python | airbyte_protocol_models_dataclasses | updated | 0.14.1 | **0.15.0** |
| python | anyio | updated | 4.7.0 | **4.9.0** |
| python | attrs | updated | 24.2.0 | **24.3.0** |
| deb | base-files | updated | 12.4+deb12u8 | **12.4+deb12u9** |
| deb | bsdutils | updated | 1:2.38.1-5+deb12u2 | **1:2.38.1-5+deb12u3** |
| python | build | updated | 0.10.0 | **1.2.2.post1** |
| python | cachetools | updated | 5.5.0 | **5.5.1** |
| python | charset-normalizer | updated | 3.4.0 | **3.4.1** |
| python | cryptography | updated | 43.0.3 | **44.0.2** |
| python | filelock | updated | 3.16.1 | **3.17.0** |
| python | h11 | updated | 0.14.0 | **0.16.0** |
| python | httpcore | updated | 1.0.7 | **1.0.9** |
| python | joblib | updated | 1.4.2 | **1.5.1** |
| deb | libblkid1 | updated | 2.38.1-5+deb12u2 | **2.38.1-5+deb12u3** |
| deb | libglib2.0-0 | updated | 2.74.6-2+deb12u4 | **2.74.6-2+deb12u5** |
| deb | libglib2.0-data | updated | 2.74.6-2+deb12u4 | **2.74.6-2+deb12u5** |
| deb | libmount1 | updated | 2.38.1-5+deb12u2 | **2.38.1-5+deb12u3** |
| deb | libopenjp2-7 | updated | 2.5.0-2 | **2.5.0-2+deb12u1** |
| deb | libsmartcols1 | updated | 2.38.1-5+deb12u2 | **2.38.1-5+deb12u3** |
| deb | libsystemd0 | updated | 252.31-1~deb12u1 | **252.33-1~deb12u1** |
| deb | libtiff6 | updated | 4.5.0-6+deb12u1 | **4.5.0-6+deb12u2** |
| deb | libudev1 | updated | 252.31-1~deb12u1 | **252.33-1~deb12u1** |
| deb | libuuid1 | updated | 2.38.1-5+deb12u2 | **2.38.1-5+deb12u3** |
| python | more-itertools | updated | 10.5.0 | **10.6.0** |
| deb | mount | updated | 2.38.1-5+deb12u2 | **2.38.1-5+deb12u3** |
| python | orjson | updated | 3.10.12 | **3.10.15** |
| python | pkginfo | updated | 1.11.2 | **1.12.0** |
| python | platformdirs | updated | 3.11.0 | **4.3.6** |
| python | poetry | updated | 1.6.1 | **1.8.4** |
| python | poetry-core | updated | 1.7.0 | **1.9.1** |
| python | poetry-plugin-export | updated | 1.6.0 | **1.8.0** |
| python | pydantic | updated | 2.10.4 | **2.10.5** |
| binary | python | updated | 3.10.14 | **3.11.11** |
| python | serpyco-rs | updated | 1.11.0 | **1.13.0** |
| python | trove-classifiers | updated | 2024.10.21.16 | **2025.1.15.22** |
| python | typing_extensions | updated | 4.12.2 | **4.13.2** |
| deb | tzdata | updated | 2024a-0+deb12u1 | **2024b-0+deb12u1** |
| python | tzdata | updated | 2024.2 | **2025.1** |
| python | urllib3 | updated | 2.2.3 | **2.3.0** |
| deb | util-linux | updated | 2.38.1-5+deb12u2 | **2.38.1-5+deb12u3** |
| deb | util-linux-extra | updated | 2.38.1-5+deb12u2 | **2.38.1-5+deb12u3** |
| python | virtualenv | updated | 20.27.1 | **20.29.1** |
| python | wheel | updated | 0.44.0 | **0.45.1** |
| python | xmltodict | updated | 0.13.0 | **0.14.2** |
| python | Pygments | added | not present | **2.19.1** |
| python | anyascii | added | not present | **0.3.2** |
| python | boltons | added | not present | **25.0.0** |
| python | fastjsonschema | added | not present | **2.21.1** |
| python | google-api-core | added | not present | **2.24.2** |
| python | google-auth | added | not present | **2.39.0** |
| python | google-cloud-secret-manager | added | not present | **2.23.3** |
| python | googleapis-common-protos | added | not present | **1.70.0** |
| python | grpc-google-iam-v1 | added | not present | **0.14.2** |
| python | grpcio | added | not present | **1.71.0** |
| python | grpcio-status | added | not present | **1.62.3** |
| python | markdown-it-py | added | not present | **3.0.0** |
| python | mdurl | added | not present | **0.1.2** |
| python | proto-plus | added | not present | **1.26.1** |
| python | protobuf | added | not present | **5.29.4** |
| python | pyasn1 | added | not present | **0.6.1** |
| python | pyasn1_modules | added | not present | **0.4.2** |
| python | rich | added | not present | **13.9.4** |
| python | rich-click | added | not present | **1.8.8** |
| python | rsa | added | not present | **4.9.1** |
| python | whenever | added | not present | **0.6.16** |
| python | Unidecode | removed | 1.3.8 | **not present** |
| python | exceptiongroup | removed | 1.2.2 | **not present** |
| python | pendulum | removed | 2.1.2 | **not present** |
| python | pytzdata | removed | 2020.1 | **not present** |
| python | tomli | removed | 2.1.0 | **not present** |

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=51438) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/airbytehq/airbyte?pullRequest=51438) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/airbytehq/airbyte?pullRequest=51438) it.</sub>

airbyteio committed:
Upgrade the base image to the latest version in metadata.yaml
PATCH bump source-woocommerce version to 0.5.5
Build source-woocommerce docker image for platform(s) linux/amd64, linux/arm64
Get dependency updates
Create or update pull request on Airbyte repository
Add changelog entry

aaronsteers commented:
Unexpected that there are no test results...

> <img width="838" alt="image" src="https://github.com/user-attachments/assets/426e55ef-b98a-4f1f-8391-07164b80ae5b" />


I think the issue is that FAST tests need to be updated to put junit results in the correct location.

aaronsteers commented:
Two failures. Both of these look okay, aka they look like false positives...

Both say 'AssertionError: Expected exception but got none.'

> <img width="740" alt="image" src="https://github.com/user-attachments/assets/b5cbf1d8-9558-44b8-b77f-06f4dfafdd3c" />

aaronsteers reviewed (approved):
Approving to force-merge. Failures appear to be false positives.

dbgold17 committed:
Merge branch 'master' into up-to-date/source-woocommerce
